src/dmd/lambdacomp.d

```diff
@@ -17,51 +29,81 @@ import dmd.statement;
 import dmd.tokens;
 import dmd.visitor;
 
+/**
+ * This enum is used to report the type of the expression
```

Comment:
`This enum is used to report` can be excised with no change in meaning to the sentence.